grails-app/views/stockMovement/_actions.gsp

```diff
@@ -6,6 +6,8 @@
         <button class="action-btn ">
             <img src="${createLinkTo(dir:'images/icons/silk',file:'bullet_arrow_down.png')}" />
         </button>
+        <g:set var="hasBeenReceived" value="${stockMovement?.shipment?.currentStatus==ShipmentStatusCode.RECEIVED}"/>
+        <g:set var="hasBeenPartiallyReceived" value="${stockMovement?.shipment?.currentStatus==ShipmentStatusCode.PARTIALLY_RECEIVED}"/>
```

Comment:
You can use `stockMovement?.shipment?.currentStatus >= ShipmentStatusCode.PARTIALLY_RECEIVED` in one variable.